src/providers/tfc/helpers.ts

```diff
@@ -46,6 +46,66 @@ export function GetRunStatusIcon(status: string): vscode.ThemeIcon {
   return new vscode.ThemeIcon('indent');
 }
 
+export function GetRunStatusMessage(status: string): string {
+  switch (status) {
+    // in progress
+    case 'pending':
+      return 'Pending';
+    case 'fetching':
+      return 'Fetching';
+    case 'queuing':
+      return 'Queuing';
+    case 'planning':
+      return 'Planning';
+    case 'cost_estimating':
+      return 'Estimating costs';
+    case 'policy_checking':
+      return 'Checking policies';
+    case 'apply_queued':
+      return 'Apply queued';
+    case 'applying':
+      return 'Applying';
+    case 'post_plan_running':
+      return 'Tasks - post-plan (running)';
+    case 'plan_queued':
+      return 'Plan queued';
+    case 'fetching_completed':
+      return 'Fetching completed';
+    case 'pre_plan_running':
+      return 'Tasks - pre-plan (running)';
+    case 'pre_plan_completed':
+      return 'Tasks - pre-plan (passed)';
+    case 'planned':
+      return 'Planned';
+    case 'cost_estimated':
+      return 'Cost estimated';
+    case 'policy_override':
+      return 'Policy override';
+    case 'policy_checked':
+      return 'Policy checked';
+    case 'confirmed':
+      return 'Confirmed';
+    case 'post_plan_completed':
+      return 'Tasks - post-plan (passed)';
+    case 'planned_and_finished':
+      return 'Planned and finished';
+    case 'policy_soft_failed':
+      return 'Policy Soft Failure';
+    case 'applied':
+      return 'Applied';
+    case 'discarded':
+      return 'Discarded';
+    case 'canceled':
+      return 'Canceled';
+    case 'force_canceled':
+      return 'Force canceled';
+    case 'errored':
+      return 'Errored';
+  }
+
+  return '';
+}
```

Comment:
I kind of wish we didn't duplicate this (and used [the same approach as we do for other enums](https://github.com/hashicorp/vscode-terraform/blob/59ec335415d577a6c613864517bac18abb10a7c2/src/terraformCloud/run.ts#L40-L60) already) but I understand the MVP nature of this. 🤷🏻‍♂️

Reply:
As well as the MVP nature, if we were to put the ThemeIcon and human readable string into an enum, we'd have to had a ThemeIcon for each item.